047-TrafficControlWithDapr/Coach/README.md

```diff
@@ -0,0 +1,26 @@
+# Coach Guide
+
+## Introduction
+Welcome to the coach's guide for the Traffic Control with Dapr What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+
+**NOTE:** If you are a Hackathon participant, this is the answer guide.  Don't cheat yourself by looking at these during the hack!  Go learn something. :)
+
+## Solutions
+- Challenge 0: **[Install tools and Azure pre-requisites](Solution-00.md)**
+   - Install the pre-requisites tools and software as well as create the Azure resources required.
+- Challenge 1: **[Run the application](Solution-01.md)**
+   - Run the Traffic Control application to make sure everything works correctly.
+- Challenge 2: **[Add Dapr service invocation](Solution-02.md)**
```

Comment:
The Coach guide ToC still has the old challenge names.  (I did see that the individual coach guide files have the correct titles.)